src/Compilers/VisualBasic/Portable/Symbols/Symbol.vb

```diff
@@ -877,7 +877,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Select Case errorInfo.Code
                     Case ERRID.ERR_UnreferencedAssemblyBase3,
                          ERRID.ERR_UnreferencedAssemblyImplements3
-                        errorInfo = ErrorFactory.ErrorInfo(ERRID.ERR_UnreferencedAssembly3, errorInfo.Arguments(0), errorInfo.Arguments(1))
+                        errorInfo = ErrorFactory.ErrorInfo(ERRID.ERR_UnreferencedAssembly3,
```

Comment:
This looks like a dead code. I do not see any place in the compiler where it can create DiagnosticInfo for ERRID.ERR_UnreferencedAssemblyBase3/ERRID.ERR_UnreferencedAssemblyImplements3. I think we should comment out the enum members (to avoid reuse of the numbers), remove this code and remove corresponding error messages from .RESX file.